### ISSUE TITLE
Problems with page titles

### ISSUE THREAD

**Issue by DavidBiddle** [alphagov/forms-prototypes#64]
There are problems with the page titles on several pages in the prototype:
- Form builder home screen: 
  - Current title: 'Form home'
  - Problem: this doesn't really sum up what this page is.
  - Suggested replacement: 'Your forms'
- Form naming screen
  - Current title: 'Create a form'
  - Problem: this might be OK, but there may be an argument for changing it to 'Name your form' or similar to better reflect what this page does.
  - Suggested replacement: keep current name or 'Name your form'
- Create/Edit question page
  - Current title: Uses the question title, e.g. 'What is your name?'
  - Problem: this doesn't indicate that this is an editing page - it would be hard to distinguish this from the corresponding preview page.
  - Suggested replacement: 'Edit question: What is your name?'
- New tab preview page:
  - Current title: 'Edit question'
  - Problem: this is completely incorrect - this isn't an editing page, and it gives no indication what page is being previewed.
  - Suggested replacement: 'Preview: What is your name?'

Arguably we should also add a '- GOV.UK Forms' suffix to all of the titles on the site - this is a common pattern.

**Comment by christophercameron-ixd** [comment]
Page titles should be the same as the H1 of the page. 

I've done this on the odd page in the prototype - I think the ideal should be: 

`[page H1] - GOV.UK Forms` if inside the creation/editing pages (within our service) 

`[page H1] - GOV.UK` if outside the creation/editing pages (for example if previewing/testing a page as the end user would see) 

... potentially include something at the start "Preview: " for example, although this might be useless information to almost all users (and would be super annoying for screen reader users, so probably a terrible idea. Also, would have massive impact on error state pages "Error: Preview: [page H1] - GOV.UK" ... ouch!)

**Comment by DavidBiddle** [comment]
Good point on the error states, hadn't thought of that!

You're right in saying that it should generally match the H1, but it's not a requirement - we can and should change it if we find that's going to be confusing.  But that's probably not necessary at this point because using the site name as you've suggested seems like a good way of disambiguating the pages.

If we find that's not enough when testing, there's scope to develop it a bit afterwards - e.g. we could use `[page H1] - GOV.UK Forms Editor`, `[page H1] - GOV.UK Forms Preview`.
